src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
```

Comment:
Remove the empty fragment tags, they are not necessary when returning a single element or component.

Reply:
+